Gems/Terrain/Code/Source/TerrainRenderer/TerrainDetailMaterialManager.cpp

```diff
@@ -98,6 +96,13 @@ namespace Terrain
         {
             return;
         }
+
+        ClipmapBoundsDescriptor desc;
+        desc.m_clipmapUpdateMultiple = 1;
+        desc.m_clipToWorldScale = DetailTextureScale;
+        desc.m_size = DetailTextureSize;
+        desc.m_worldSpaceCenter = AZ::Vector2(AZStd::numeric_limits<float>::max(), 0.0f);
```

Comment:
Why does this start at max for X, but 0 in Y? A comment explaining this would be nice.

Reply:
I'll comment. I could set Y to max as well but it doesn't really matter, this just needs to be some position that a "real" camera would never be in, so just setting one dimension to max is enough.